internal/inbox/channel/telegram/telegram.go

```
@@ -0,0 +1,516 @@
+	return ""
+}
+
+func (t *Telegram) Channel() string {
+	return ChannelTelegram
+}
+
+func (t *Telegram) SetWebhook(url string) error {
+	apiURL := fmt.Sprintf("%s%s/setWebhook", telegramAPIBase, t.config.BotToken)
+	payload, _ := json.Marshal(map[string]string{"url": url})
+
+	resp, err := t.httpClient.Post(apiURL, "application/json", bytes.NewReader(payload))
```

golangci-lint (2.12.2)

[error] (*net/http.Client).Post must not be called. use (*net/http.Client).Do(*http.Request)

(noctx)